src/langcheck/metrics/eval_clients/_openai.py

```diff
@@ -2,12 +2,14 @@
 
```

Comment:
Let's also bump the version in this PR!

https://langcheck.readthedocs.io/en/latest/contributing.html#publishing